### ISSUE TITLE
install: omitting directory '/home/chronos/user/Downloads/crouton'

### ISSUE THREAD

**Issue by rmtx83** [dnschneid/crouton#4735]
```


sudo install -Dt /usr/local/bin -m 755 ~/Downloads/crouton
 install: omitting directory '/home/chronos/user/Downloads/crouton'
please help ran this command and this happend every time cant find any results on how to fix

**Comment by TomTravis** [comment]
install: omitting directory '/home/chronos/user/Downloads/crouton'
please help ran this command and this happend every time cant find any results on how to fix
are you using a guest account on your chromebook? user is the guest account  -- deletes every thing when you logout, use a normal account you need a internet connection

**Comment by BetaPictoris** [comment]
@TomTravis 
> are you using a guest account on your chromebook? user is the guest account -- deletes every thing when you logout, use a normal account you need a internet connection

In my experience a guest account works better than, or at least as well as, a Google account when installing Crouton.

**Comment by TomTravis** [comment]
yes In my experience a guest account works better than, or at least as well as, a Google account when installing Crouton.
as long as you install in a place that does not get deleted when you logout The Downloads user dir is deleted and new for each guest so crouton is missing?

**Comment by BetaPictoris** [comment]
@TomTravis 
> as long as you install in a place that does not get deleted when you logout The Downloads user dir is deleted and new for each guest so crouton is missing?

To the best of my knowledge it is rather difficult to install Crouton outside of a directory that gets deleted when exiting the guest account.

@rmtx83, it looks like you are trying to install a directory. Can you check to see if you have a file named “crouton” in your downloads directory? If it’s a directory delete it and download the file from [here](https://goo.gl/fd3zc) and try installing it again.

**Comment by Githubant123** [comment]
hello
